src/main/java/reposense/git/GitClone.java

```diff
@@ -62,6 +62,10 @@ public static void clone(RepoConfiguration repoConfig)
         }
     }
 
+    /**
+     * Clones a repo given the location and folder name.
+     * @throws IOException
```

Comment:
Should include description when adding a tag. There is no point adding one, if you're not providing useful information, as its already included in the function header.